feature-utils/poly-look/src/react-components/buttons/button.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import "./button.css";
+
+/**
+ * Basic button.
+ * In addition to `label` any valid button HTML attributes are allowed.
+ * @param {Object} props
+ * @param {string} [props.label] - Button text
+ */
+
+const Button = ({ label, ...otherProps }) => {
+  return (
+    <button
+      {...otherProps}
+      className={`poly-button ${otherProps.className || ""}`}
```

Comment:
So `otherProps` will contain `Button` attributes? And why is the class name used for the class here?

Reply:
Also, the only classes defined above are "active" and "disabled". `className`  will have one of those two values?

Reply:
Initially I didn't wanted to allow any other classes, but after I've used this component in `ClickableCard` I've realized that if I want to keep the refactoring in other places to a minimum some flexibility should be allowed; that's why the final value for `className` will be `poly-button` plus whatever classes you might want to pass down for customization.

Now, for this button we have 3 states: normal, pressed (active) and disabled (disabled).  And we can achieve that for free with CSS without having to add any code.

Reply:
A nit, though: If there's no other class, the class will be `polyButton `, with a whitespace at the end. Not a big deal, but easily fixable, too. Good call with the no-code states, too :+1:

Reply:
I've fixed the whitespace.